test/__mocks__/text.mock.ts

```diff
@@ -152,6 +152,7 @@ export const INFORMATION_SHOWN_ON_THE_PUBLIC_REGISTER = "What information we’l
 export const NOT_SHOW_BENEFICIAL_OWNER_INFORMATION_ON_PUBLIC_REGISTER = "We will not show the beneficial owner’s home address unless it is the same as their correspondence address. We’ll only show the month and year of their date of birth.";
 export const NOT_SHOW_MANAGING_OFFICER_INFORMATION_ON_PUBLIC_REGISTER = "We will not show the managing officer’s home address unless it is the same as their correspondence address. We’ll only show the month and year of their date of birth.";
 export const SHOW_INFORMATION_ON_PUBLIC_REGISTER = "We will show all the information you provide on this screen on the public Register of Overseas Entities.";
+export const SHOW_CMO_INFORMATION_ON_PUBLIC_REGISTER = "We will show all the other information you provide on this screen on the public Register of Overseas Entities.";
```

Comment:
Already present, so will remove and change for the const already there then introduce NOT_SHOW_NAME_OR_EMAIL_ON_PUBLIC_REGISTER = "We will not show the name or email address of the person who can be contacted about this managing officer.";
which I will add to the tests.

Reply:
Updated this to use existing line and test the new line.